LDK/resources/web/LDK/plugin/UserEditableCombo.js

```diff
@@ -81,9 +81,11 @@ Ext4.define('LDK.plugin.UserEditableCombo', {
             },
 
             setValue: function(val){
-                this.addValueIfNeeded(val);
+                if (val) {
```

Comment:
Also, doesnt UserEditableCombo.addValueIfNeeded() already abort if the input is null?

Further, i dont think "if (val)" is a good check. What if the input value is zero or false? Those are theoretically legitimate values for a combo. UserEditableCombo.addValueIfNeeded already uses Ext4.isEmpty(), which should handle this.